src/mibase.ts

```diff
@@ -776,7 +776,80 @@ export class MI2DebugSession extends DebugSession {
 			this.sourceFileMap = new SourceFileMap(configMap, fallbackGDB);
 		}
 	}
+	public async createIntegratedTerminalLinux(args:DebugProtocol.RunInTerminalRequestArguments) {
+		const mkdirAsync = fs.promises.mkdir;
+		const mkdtempAsync = async (tempDir: string, prefix: string): Promise<string> => {
+			const name = `${prefix}-${Date.now()}-${Math.floor(Math.random() * 1e9)}`;
+			const newDirPath = `${tempDir}/${name}`;
 
+			try {
+				await mkdirAsync(newDirPath, { recursive: true });
+				return newDirPath;
+			} catch (err) {
+				throw new Error(`Error creating temp directory: ${err.message}`);
+			}
+		};
+		const ttyTmpDir = await mkdtempAsync(os.tmpdir(), 'debug');
+
+		(async () => {
+			try {
+				fs.writeFileSync(
+					`${ttyTmpDir}/get-tty`,
+					`#!/usr/bin/env sh
+					# reset terminal to clear any previous states
+					reset
```

Comment:
This uses the shell's init commands, no?

Should this be suppressed? Should there be an option to add extra commands?